frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -264,8 +270,9 @@ export const getMarkerIcon = (feature: ICluster, selected?: boolean) => {
           return landErpIconSelect;
       }
     } else if (
-      selectedClassificationId === Classifications.SurplusActive ||
-      selectedClassificationId === Classifications.SurplusEncumbered
+      (selectedClassificationId === Classifications.SurplusActive ||
+        selectedClassificationId === Classifications.SurplusEncumbered) &&
+      projectNumbers.length
```

Comment:
Unfortunately I still think this is not quite right because projectNumbers may have one or more draft projects which shouldn't change the display of the parcel.

My suggestion would be to see if we can use the workflowcode similar to how erp is working, or if not just check for the presence of SPP in the projectNumbers array.

Reply:
Ahh ok, makes sense. Will adjust - thanks!